Wox/SettingWindow.xaml

```diff
@@ -312,6 +314,22 @@
                                 </DataTemplate>
                             </ComboBox.ItemTemplate>
                         </ComboBox>
+                        
+                        <ComboBox ItemsSource="{Binding SelectedResultFont.FamilyTypefaces}"
```

Comment:
![image](https://user-images.githubusercontent.com/1169912/80274629-601c9800-870e-11ea-9c94-532621976236.png)
there should be a description for this settings